### PR TITLE
Labbranch

### PR DESCRIPTION
**Pull request by MrDouglas13**
Fix this pls

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (msbence) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: noice solution

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (msbence) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: ivós cs edition:
Exception up = new IllegalArgumentException("");
throw up;

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
...
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
-    double r = generator.nextDouble();
+    double r = this.generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (msbence) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: u bastard

**Comment by MrDouglas13** [comment]
all fixed

**Pull request merged**